src/lib/components/wizard.component.spec.ts

```diff
@@ -86,16 +84,13 @@ describe('WizardComponent', () => {
     expect(wizardTestFixture.debugElement.query(By.css('aw-wizard > :first-child')).name).toBe('aw-wizard-navigation-bar');
     expect(wizardTestFixture.debugElement.query(By.css('aw-wizard > :last-child')).name).toBe('div');
 
-    expect(navBar.classes).toEqual({
```

Comment:
It no longer need to have all classes with false and true, but only the classes that needed